lumigator/python/mzai/jobs/inference/inference.py

```diff
@@ -0,0 +1,161 @@
+"""python job to run batch inference"""
+
+import argparse
+import functools
+import json
+import os
+import time
+from collections.abc import Iterable
+from pathlib import Path
+
+import s3fs
+from box import Box
+from datasets import load_from_disk
+from loguru import logger
+from model_clients import (
+    BaseModelClient,
+    MistralModelClient,
+    OpenAIModelClient,
+)
+
+# SummarizationPipelineModelClient,
```

Comment:
can we remove?

Reply:
Yes! I was planning to also have transformers code originally but if we go the vLLM way we won't need this